build-decision/src/build_decision/cron/decision.py

```diff
@@ -39,12 +40,20 @@ def run_decision_task(job_name, job, *, repository, push_info, dry_run):
 
     arguments = make_arguments(job)
 
+    cron_input = {}
+    if job.get("include-cron-input") and "HOOK_PAYLOAD" in os.environ:
```

Comment:
I added `include-cron-input` to make it consistent with the "action" type, but I'm wondering if we shouldn't just always forward "HOOK_PAYLOAD" to the render context. I can't really think of any risks with doing that

Reply:
Maybe I'll just leave it in-place for now to be consistent (and out of an abundance of caution). I'll file an issue to look into removing both of them later.